### DISCUSSION TITLE
Only three blocks can be used after I installed the code

### DISCUSSION THREAD

**Discussion by pulsegame** [GrapesJS/grapesjs#6043] [Q&A]
Dear GrapesJS team,
I love your project really, but I only can see three block as screenshot when I install the demo, could you help me which how to enable more blocks which showed on live demo, thanks.

![20240807-172807](https://github.com/user-attachments/assets/df08e948-cdc7-4020-b6b7-4addafce0865)

**Comment by artf** [comment]
I'd recommend starting from the beginning: https://grapesjs.com/docs/getting-started.html
